src/client/Client.js

```diff
@@ -356,21 +356,35 @@ class Client extends BaseClient {
 
   /**
    * Generates a link that can be used to invite the bot to a guild.
-   * @param {PermissionResolvable} [permissions] Permissions to request
+   * @param {InviteGenerationOptions|PermissionResolvable} [options] Permissions to request
    * @returns {Promise<string>}
    * @example
    * client.generateInvite(['SEND_MESSAGES', 'MANAGE_GUILD', 'MENTION_EVERYONE'])
    *   .then(link => console.log(`Generated bot invite link: ${link}`))
    *   .catch(console.error);
```

Comment:
The example points an usage of this method that is now deprecated.

Reply:
O, should i update the example to use all 3 options or multiple examples, one for permissions and 1 for the guild option?

Reply:
Only one, similar to [`TextChannel#edit()`](https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=edit), which accepts multiple options in an object but shows only one in the example.